app/views/coronavirus_landing_page/components/landing_page/_icon-face-cover.svg

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<svg class="covid__instruction-icon" width="45" height="45" version="1.1" viewBox="0 0 400 400" xml:space="preserve" xmlns="http://www.w3.org/2000/svg">
+	<path d="m93.03 249.99 11.41-2c-8.7-16.67-16.13-38.68-17.56-54.84-1.21-13.68-0.9-25.08 3.25-35.05l-12.84 2.26c-24.41 4.28-40.79 27.63-36.5 52.04l0.2 1.1c4.28 24.4 27.63 40.78 52.04 36.49m266.18-37.59c4.28-24.41-12.09-47.76-36.5-52.04l-12.84-2.26c4.16 9.96 4.46 21.37 3.26 35.05-1.43 16.16-8.87 38.17-17.56 54.84l11.4 2c24.41 4.28 47.77-12.1 52.05-36.5l0.19-1.09zm9.99 1.75-0.2 1.1c-5.25 29.97-33.82 50-63.79 44.73l-15.04-2.63c-3.42 5.32-6.88 9.56-10.13 12.09-13.53 10.52-46.78 37.58-80.05 36.45-33.26 1.13-66.52-25.93-80.05-36.45-3.24-2.52-6.7-6.77-10.13-12.09l-15.04 2.63c-29.97 5.26-58.53-14.76-63.79-44.73l-0.2-1.1c-5.26-29.97 14.77-58.53 44.74-63.79l21.83-3.83c3.55-4.09 8.11-7.94 13.93-11.64 31.2-18.78 69.15-39.08 88.7-40.2 19.54 1.13 57.5 21.42 88.7 40.2 5.82 3.7 10.38 7.55 13.93 11.64l21.83 3.83c29.99 5.26 50.02 33.82 44.76 63.79m30.8-13.85c0-110.46-89.54-200-200-200s-200 89.54-200 200 89.54 200 200 200 200-89.54 200-200"/>
```

Comment:
Could this (and other SVGs in this commit) be compressed more?

Reply:
Already compressed; I removed the new lines though, they seem to save up to 3%.